package.json

```diff
@@ -67,6 +67,7 @@
     "cors": "^2.8.5",
     "dayjs": "^1.8.34",
     "dotenv": "^8.2.0",
+    "env": "0.0.2",
```

Comment:
you do not need this env inside package.json.
you shouldn't add all changes to master just add your file !

Reply:
i guess it's from when we did npm install env together on a call by mistake when it should be dotenv package